### ISSUE TITLE
呜呜 初中学生党买不起女装

### ISSUE THREAD

**Issue by BreakingLead** [komeiji-satori/Dress#1283]
qaq

**Comment by TheWhistlers** [comment]
同啊，难受啊马飞

**Comment by itqihou** [comment]
Don't tell me you are Chinese.You are not Chinese, are you?Chinese people would rather die than pretend to be women. How can they be as weak as you.

**Comment by Angelic47** [comment]
请注意发言时尊重他人，这里禁止任何形式的国籍、地域、性别等歧视性言论。每个人的性格是不一样的，请予以尊重，尤其不要用带有性别刻板印象和地域刻板印象的思想去评价他人。这在任何地方都是侵犯他人名誉的表现。
你可以不赞同别人的想法，但不可以用人身攻击的方式侵犯他人的合法权利。
此外，给女性直接打上“weak”的标签，并以此来证明其他论点，这本身就是对所有人的歧视性思维。

**Comment by itqihou** [comment]
> 

Yes, sir or madam. Oh, I see. I realized my mistake. Please let me apologize to you!

**Comment by iamchenguanxi666** [comment]
my god 初中生都知道这个网站我今天才知道

**Comment by Kamisato-Ayaka-233** [comment]
我也初中生哈哈
-`虽然并不想女装`
